src/Meshes/geometry.ts

```diff
@@ -1208,31 +1209,61 @@ export class Geometry implements IGetSetVerticesData {
 
             if (binaryInfo.uvsAttrDesc && binaryInfo.uvsAttrDesc.count > 0) {
                 var uvsData = new Float32Array(parsedGeometry, binaryInfo.uvsAttrDesc.offset, binaryInfo.uvsAttrDesc.count);
+                if (CompatibilityOptions.UseOpenGLOrientationForUV) {
+                    for (var index = 1; index < uvsData.length; index += 2) {
+                        uvsData[index] *= -1;

```

Comment:
Why not `1-uvsData[index]`? I think it's not the same thing than `*= -1` if the address mode is CLAMP.

For eg, if v=0.3, the current code will compute -0.3 and in CLAMP mode you will get 0. With `1-v` you get 0.7 and still get 0.7 in CLAMP mode.

Same comment for all the loops below.